go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/openshift/installer
 
-go 1.20
+go 1.21
```

Comment:
Is this required by some specific dependency?

Reply:
Just pulled from master and now it is synced